docs/ruleset_functions/is_fluid_loop_attached_to_boiler(hvac_b.id).md

```diff
@@ -0,0 +1,24 @@
+# is_fluid_loop_attached_to_boiler  
+
+**Description:** Returns TRUE if the fluid loop associated with the heating system associated with the HVAC system is attached to a boiler. Returns FALSE if this is not the case.   
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the fluid loop associated with the heating system associated with the HVAC system is attached to a boiler.   
+- **hvac_b.id**: The id of the hvac system to evaluate.  
+
+**Returns:**  
+- **is_fluid_loop_attached_to_boiler**: Returns TRUE if the fluid loop associated with the heating system associated with the HVAC system is attached to a boiler. Returns FALSE if this is not the case.   
+ 
+**Function Call:** None  
+
```

Comment:
i believe this function might need an applicability criteria for checking if is fluid loop = true?
would it make sense to make self contained functions which check if HVAC system is connected to fluid loop and if the fluid loop is connected to boiler/PHW
then there wouldn't be a need for the applicability criteria

Reply:
@supriyagoel This applicability check is already done before this function is called in https://github.com/pnnl/ruleset-checking-tool/pull/480/files . If there is not a fluid loop then the other function would not call this one.